pkg/fleet-manager/manifests/plugins/velero.yaml

```diff
@@ -0,0 +1,18 @@
+type: default
+repo: https://vmware-tanzu.github.io/helm-charts
+name: velero
+version: 5.0.2
+targetNamespace: velero
+values:
+  image:
+    repository: velero/velero
+    tag: v1.11.1
+  deployNodeAgent: true
+  defaultVolumesToFsBackup: true
+  snapshotsEnabled: false
+  initContainers:
+    - image: velero/velero-plugin-for-aws:v1.7.1
```

Comment:
what does this init container do

Reply:
The initContainers configuration is the tool for communicating with AWS S3 or similar storage solutions, like the Minio setup we are using. 
see https://github.com/vmware-tanzu/helm-charts/blob/7564a60a891199aff32dc4f34ca9f1e882e18e59/charts/velero/values.yaml#L58-L60